servers/audio/audio_rb_resampler.h

```diff
@@ -86,7 +86,7 @@ struct AudioRBResampler {
 		} else if (w < r) {
 			space = r - w - 1;
 		} else {
-			space = (rb_len - r) + w - 1;
+			space = rb_len - w + r - 1;
```

Comment:
```suggestion
			space = (rb_len - w) + (r - 1);
```
Can you check if this has the same meaning?

Reply:
Of course, it has the same meaning. Are you suggesting to put the parenthesis?

Reply:
Yes